domain/entity/latest_entry.go

```diff
@@ -0,0 +1,90 @@
+package entity
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/appengine/datastore"
+)
+
+const (
+	LatestEntryCodeTamai    = "tamai-sd"
+	LatestEntryCodeMomota   = "momota-sd"
+	LatestEntryCodeAriyasu  = "ariyasu-sd"
+	LatestEntryCodeSasaki   = "sasaki-sd"
+	LatestEntryCodeTakagi   = "takagi-sd"
+	LatestEntryCodeHappyclo = "happyclo"
+)
+
+type (
+	LatestEntry struct {
+		ID        string `datastore:"-" goon:"id"`
+		Code      string
+		URL       string
+		CreatedAt time.Time
+		UpdatedAt time.Time
+	}
+
+	LatestEntryRepository interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type LatestEntryRepository should have comment or be unexported